search/search.py

```diff
@@ -11,7 +14,29 @@ def get_search_blueprint() -> Blueprint:
 
     @blueprint.route('/search', methods=['GET'])
     def search() -> Response:
+
+        # get query string and decode bytes. If present add it to es query
+        kwargs = {}
+        query_string = request.query_string.decode('utf-8')
+        if query_string:
+            kwargs.update(dict(q=query_string))
+
+        # get all documents or query for search term
+        search = Elasticsearch(hosts=ELASTICSEARCH_HOSTS)
+        # pylint: disable=unexpected-keyword-arg
+        hits = search.search(index='_all', filter_path=['hits.hits._source'], **kwargs)
```

Comment:
there is no explicit ordering specified in this call (will be by score?), what does the driver guarantee in this case?

Reply:
The only requirement/task in issue 107 is `Return the items in an unordered (but ideally consistent) unpaginated list`

Reply:
Yes, so is (some, doesn't matter which) ordering guaranteed across requests? SQL doesn't without an explicit `ORDER BY`, not sure in ElasticSearch. The [6.6 documentation on sorting](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-sort.html) talks about special `_score` or `_doc` parameters that could be helpful, but I haven't seen a default.